src/ios/SimpleCameraPreview.m

```diff
@@ -78,11 +82,25 @@ - (void) enable:(CDVInvokedUrlCommand*)command {
     
     [self.sessionManager setupSession:@"back" completion:^(BOOL started) {
         dispatch_async(dispatch_get_main_queue(), ^{
-            [self.commandDelegate sendPluginResult:[CDVPluginResult resultWithStatus:CDVCommandStatus_OK] callbackId:command.callbackId];
+            CDVPluginResult *pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK];
+            [pluginResult setKeepCallbackAsBool:true];
+            [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
```

Comment:
why do we need this part ? @parveshneedhoo

Reply:
If we dont use `setKeepCallbackAsBool:True` in the enable method(from which the observer are listened) the plugin results are not returned.